packages/server/__tests__/db/db.test.js

```diff
@@ -20,23 +20,22 @@ describe('db', () => {
 
     context('getGrantsInterested', () => {
         it('gets the most recent interested grant', async () => {
-            const result = await db.getGrantsInterested({ perPage: 1, currentPage: 1 });
-            // console.log(fixtures.grantsInterested.entry2);
-            expect(result.rows[0]).to.have.property('grant_id').with.lengthOf(6);
-            expect(result.rows[0].grant_id).to.equal(fixtures.grantsInterested.entry2.grant_id);
+            const rows = await db.getGrantsInterested({ agencyId: fixtures.users.staffUser.agency_id, perPage: 1, currentPage: 1 });
+            expect(rows[0]).to.have.property('grant_id').with.lengthOf(6);
+            expect(rows[0].grant_id).to.equal(fixtures.grantsInterested.entry2.grant_id);
             // in the grants interested table the grant with the most recent created_at has the grant id of 335255
-            expect(result.rows[0]).to.have.property('grant_id').equal('335255');
+            expect(rows[0]).to.have.property('grant_id').equal('335255');
         });
         it('gets the two most recent interested grants', async () => {
             // testing pagination
-            const result = await db.getGrantsInterested({ perPage: 2, currentPage: 1 });
-            expect(result.rows).to.have.lengthOf(2);
+            const rows = await db.getGrantsInterested({ agencyId: fixtures.users.staffUser.agency_id, perPage: 2, currentPage: 1 });
+            expect(rows).to.have.lengthOf(2);
```

Comment:
Can these tests that look at rows.length check the actual IDs of grants returned?

Reply:
I changed this one to check the date ordering, so the created_at values, since they reference the same grant id. The tests below are actually stats endpoints so they can only verify the count as its all that is returned by the api.